script.js

```diff
@@ -0,0 +1,257 @@
+
+////////////1///////////////
+
+let result = prompt ('Ваше имя?', '');
+alert (`Ваше имя ${result}`);
+
+const sayHello = function(name) {
+    if (name === "Mark") {
+        return `Hi, ${name}`;
+    }
+        return `Hello, ${name}`;
+    
+}
+
+
+console.log(sayHello("Oleg"));
+console.log(sayHello("Victor"));
+console.log(sayHello("Mark"));
+
+
+
+const sayHello = (name) => {
+    if (name === "Mark") {
+        return `Hi, ${name}`;
+    }
+        return `Hello, ${name}`;
+    
+}
+
+
+//////////////2///////////////////
+
+console.log(sayHello("Oleg"));
+console.log(sayHello("Victor"));
+console.log(sayHello("Mark"));
+
+
+const hypotenuse = function (n ,m) {
+    if (typeof n === "number" && typeof m === "number") {
+        return ((n**2 + m**2)**(1/2));
+    }
+}
+
+console.log(hypotenuse(3, 4));
+
+const hypotenuse = (n ,m) => {
+    if (typeof n === "number" && typeof m === "number") {
+        return ((n**2 + m**2)**(1/2));
+    }
+}
+
+console.log(hypotenuse(3, 4));
+
+
+
+//////////////3///////////////////
+
+
+
+const minValue = function (value1 ,value2) {
+    if (typeof value1 === "number" && typeof value2 === "number" && value1<value2) {
+        return value1;
+    }
+
+    if (typeof value1 === "number" && typeof value2 === "number" && value1>value2) {
+        return value2;
+    }
+}
+
+console.log(minValue(2, 5));
+console.log(minValue(3, -1));
+console.log(minValue(1, 1));
+
+const minValue = (value1 ,value2) {
+    if (typeof value1 === "number" && typeof value2 === "number" && value1<value2) {
+        return value1;
+    }
+
+    if (typeof value1 === "number" && typeof value2 === "number" && value1>value2) {
+        return value2;
+    }
+}
+
+console.log(minValue(2, 5));
+console.log(minValue(3, -1));
+console.log(minValue(1, 1));
+
+
+//////////////4///////////////////
+
+
+const isEven = function (value) {
+    if (typeof value === "number" && value % 2 == 0) {
+        return true;
+    }
+    return false;
+
```

Comment:
`value % 2 === 0` return `true` or `false` depends on parity of entered number.
I mean that you can just use single return `return value % 2 === 0` and it will be enough for this task.